Packs/GoogleCloudCompute/Integrations/GoogleCloudCompute/GoogleCloudCompute.py

```diff
@@ -913,7 +928,10 @@ def get_instance(args):
     parameter: (string) instance
         Name of the instance scoping this request.
     """
-    project = SERVICE_ACT_PROJECT_ID
+    project = args.get('project_id')
+    if not project:
+        project = SERVICE_ACT_PROJECT_ID
```

Comment:
Extract to method: `get_project_id(args)`
These line repeat themselves several time in the code.